app/services/action_queue.rb

```diff
@@ -119,6 +119,8 @@ class DirectDebitAction
     include Donatable
 
     def payload
+      #TODO: This really needs to be handled in a less hackish solution once we refactor.
+      @action.form_data.merge!({action_express_donation: 0})
```

Comment:
Getting these classes to play nicely with additions is really pretty difficult - I needed to append the custom field to the form params this way, which isn't great. This will actually be a bit tidier once we send this parameter from the client.